AgentAssistAngular/src/common/components/overlaysearch/overlaysearch.component.ts

```diff
@@ -91,6 +91,7 @@ export class OverlaysearchComponent implements OnInit {
   }
 
   emitSearchRequest(searchObj, isSearchFlag, faqToggle = false) {
+    console.log('Sandeep Test1: ', searchObj, isSearchFlag);
```

Comment:
@SandeepBa-Kore remove this console, and I'm seeing unnecessary modifications in this file.